grml-debootstrap

```diff
@@ -1732,6 +1825,11 @@ iface ${interface} inet dhcp
     einfo "Copying /etc/network/interfaces from host to target system"
     mkdir -p "${MNTPOINT}/etc/network"
     cp ${VERBOSE:+-v} /etc/network/interfaces "${MNTPOINT}/etc/network/interfaces"
+    if [ -r /etc/network/interfaces.d ] ; then
```

Comment:
this should land in a separate PR

Reply:
Good point, this is sort of a quirk of the fact that I'm using one of the raspi.debian.net images as my host system and it stores its configuration in interfaces.d, not directly related to the kind of image I'm building.

Reply:
Removed for now, will make a second PR for it.